File: test/aidbox_sdk/snapshots/dotnet/hl7-fhir-r4-core/OperationOutcome.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class OperationOutcome : DomainResource, IResource
{
    public required OperationOutcome_Issue[] Issue { get; set; }

    public class OperationOutcome_Issue : BackboneElement
    {
        public required string Code { get; set; }
        public Base.CodeableConcept? Details { get; set; }
        public string[]? Location { get; set; }
        public required string Severity { get; set; }
        public string[]? Expression { get; set; }
        public string? Diagnostics { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/OperationOutcome.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import DomainResource
from base import BackboneElement

@dataclass(kw_only=True)
class OperationOutcomeIssue(BackboneElement):
    code: str
    details: Optional[CodeableConcept] = None
    location: Optional[List[str]] = field(default_factory=list)
    severity: str
    expression: Optional[List[str]] = field(default_factory=list)
    diagnostics: Optional[str] = None

@dataclass(kw_only=True)
class OperationOutcome(DomainResource):
    issue: list[OperationOutcomeIssue] = field(default_factory=list)